README.md

```diff
@@ -34,14 +34,14 @@ Recurrence relation:
 
        1     if n <= 1  
 T(n){
-       3T(n/3) + n   if n >1
+       3T(n/3)   if n >1
```

Comment:
There's something missing here -- how much work is required to combine the results of the recursive calls?